### ISSUE TITLE
fit the screen

### ISSUE THREAD

**Issue by X-SLAYER** [AdityaMulgundkar/wallpaper_manager#19]
hello, I used the package but when I set the image its doesn't fit the screen correctly I used an image from Zedge that already fitted my screen but with the plugin, I didn't get the perfect cover.

**Comment by hizmetciAli** [comment]
did u solve the problem ?

**Comment by X-SLAYER** [comment]
no, I didn't because I didn't focus on the project I will try to find a solution asap

**Comment by hizmetciAli** [comment]
Okey , thx  Windows 10 için Posta ile gönderildi Kimden: Iheb BrikiGönderilme: 10 Mart 2021 Çarşamba 12:01Kime: AdityaMulgundkar/wallpaper_managerBilgi: hizmetciAli; CommentKonu: Re: [AdityaMulgundkar/wallpaper_manager] fit the screen (#19) no, I didn't because I didn't focus on the project I will try to find a solution asap—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
